lms/data_tasks/report/create_from_scratch/02_entities/03_groups/03_groups/01_create_view.sql

```diff
@@ -0,0 +1,39 @@
+DROP MATERIALIZED VIEW IF EXISTS report.groups CASCADE;
+
+-- A view with our concept of a unique group in reporting, which happens to
+-- mostly conform to an H group with extra metadata. This differs from the
+-- `report.raw_group` it is based on by adding `parent_id` information.
+
+CREATE MATERIALIZED VIEW report.groups AS (
+    SELECT
+        raw_groups.id,
+        parent_ids.parent_id,
+        raw_groups.authority_provided_id,
+        raw_groups.name,
+        raw_groups.group_type,
+        raw_groups.created
```

Comment:
We could do `raw_groups.*`, but I prefer spelling out the columns and it allows us to put the id and parent_id next to each other.